loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -594,6 +596,70 @@ module BABYLON.GLTF2 {
             var material = getMaterial(runtime, primitive.material);
             multiMat.subMaterials.push(material);
 
+            // Morph Targets
+            if (primitive.targets !== undefined) {
+                morphTargetManager = new BABYLON.MorphTargetManager();
+
+                for (var index2 = 0; index2 < primitive.targets.length; index2++) {
+                    var target = primitive.targets[index2];
+
+                    var weight = 0.0;
+                    if (node.weights !== undefined) {
+                        weight = node.weights[index2];
+                    }
+                    else if (mesh.weights !== undefined) {
+                        weight = mesh.weights[index2];
+                    }
+
+                    var morph = new BABYLON.MorphTarget("morph" + index2, weight);
+
+                    for (var semantic in target) {
+                        // Link accessor and buffer view
+                        accessor = runtime.gltf.accessors[target[semantic]];
+                        buffer = GLTFUtils.GetBufferFromAccessor(runtime, accessor);
+
+                        // glTF stores morph target information as deltas
+                        // while babylon.js expects the final data.
+                        // As a result we have to add the original data to the delta to calculate
+                        // the final data.
+                        if (semantic.toUpperCase() === "NORMAL") {

```

Comment:
The [morph target section](https://github.com/KhronosGroup/glTF/tree/2.0/specification/2.0#morph-targets) explicit states what values are supported. The [schema/property reference](https://github.com/KhronosGroup/glTF/tree/2.0/specification/2.0#primitivetargets) also explicitly calls this out. All values must be uppercase.